src/pages/variantsSearchPage/WorkBench.tsx

```diff
@@ -125,29 +134,61 @@ const WorkBench = (props: Props) => {
               </Space>
             )}
             {showProgress(status) && <IndeterminateProgress />}
-            {error && (
-              <Alert
-                className={style.alert}
-                message={
-                  <>
-                    <span>{'We were unable to complete this operation. '}</span>
-                    <a href={'mailto:support@kidsfirstdrc.org'}>Contact support</a>
-                    <span>{' if the issue persists'}</span>
-                  </>
-                }
-                type="error"
-                action={
-                  <Button type="primary" danger onClick={() => onReInitializeState()}>
-                    Try again
-                  </Button>
-                }
-              />
-            )}
+            {error && displayError(error, loggedInUser, onReInitializeState)}
           </div>
         </StackLayout>
       </StackLayout>
     </Card>
   );
 };
 
+const displayError = (error: any, loggedInUser: LoggedInUser, onReInitializeState: () => void) => {
```

Comment:
the type of error is defined in `store/WorkBenchState.ts` We should avoid any except when we have good reason to and document it in the code.